examples/todoapp/lib/base/utils/handle_error_mixin.dart

```diff
@@ -0,0 +1,23 @@
+import '../models/errors/error_model.dart';
+
+mixin ErrorHandlingMixin {
```

Comment:
Please be more specific on the mixin name. Now is too generic

Reply:
Done